app/views/listings/delete_confirm.html.erb

```diff
@@ -1,24 +1,23 @@
-<%= javascript_include_tag "application" %>
-
-<div class="crayons-layout crayons-layout--limited-l gap-0">
+<main class="crayons-layout crayons-layout--limited-l gap-0">
```

Comment:
Nice catch! A small suggestion so that the skip link works on this page too
```suggestion
<main  id="main-content" class="crayons-layout crayons-layout--limited-l gap-0">
```

Reply:
thank for spotting this!